build/contracts/MetaCoin.json

```diff
@@ -2508,5 +2508,5 @@
     }
```

Comment:
You're probably going to want to put these two contract files into your gitignore, since it seems like they're auto-generated by the code and get automatically updated with a timestamp every time a change is made to the system.